Images in page bundles

bdillahu opened the pull request:
Here's my attempts (it worked for me) to tweak the code back to supporting images with relative links in page bundles instead of a central "assets" directory.

bdillahu committed:
Update writer.js

Putting image subdirectory code back

bdillahu committed:
tweaks to handle page bundles with relative links to images

bdillahu committed:
tweaks to handle page bundles with relative links to images

rufuspollock commented:
@bdillahu sorry for slow reply - reading the code i wasn't clear if you changed the default behaviour or add an option to take this different approach.

bdillahu commented:
> @bdillahu sorry for slow reply - reading the code i wasn't clear if you changed the default behaviour or add an option to take this different approach.

For my work I just changed the default behavior - (given this was a while back, my memory is a bit fuzzy) - As I recall, it appeared to have at least at one time supported page bundles, but parts didn't work any more, so I fixed them (at least enough to get the migration I was working on to complete :-)).
